diff --git a/include/configs/zynq_common.h b/include/configs/zynq_common.h
index f8ff0ac..477c9b6 100644
--- a/include/configs/zynq_common.h
+++ b/include/configs/zynq_common.h
@@ -290,8 +290,7 @@
 	"jtagboot=echo TFTPing Linux to RAM... && " \
 		"tftp 0x3000000 ${kernel_image} && " \
 		"tftp 0x2A00000 ${devicetree_image} && " \
-		"tftp 0x2000000 ${ramdisk_image} && " \
-		"bootm 0x3000000 0x2000000 0x2A00000\0"
+		"bootm 0x3000000 - 0x2A00000\0"
 
 /* default boot is according to the bootmode switch settings */
 #define CONFIG_BOOTCOMMAND		"run $modeboot"
diff --git a/include/configs/zynq_zed.h b/include/configs/zynq_zed.h
index e4ba3f0..7b87f71 100644
--- a/include/configs/zynq_zed.h
+++ b/include/configs/zynq_zed.h
@@ -26,9 +26,9 @@
 
 #define CONFIG_SYS_NO_FLASH
 
-#define CONFIG_ZYNQ_SDHCI0
-#define CONFIG_ZYNQ_QSPI
-#define CONFIG_ZYNQ_BOOT_FREEBSD
+//#define CONFIG_ZYNQ_SDHCI0
+//#define CONFIG_ZYNQ_QSPI
+//#define CONFIG_ZYNQ_BOOT_FREEBSD
 
 #include <configs/zynq_common.h>
 
